### PR TITLE
[SIG-3040] Styling "Thank you" page

### PR DESCRIPTION
**Pull request by janjaap**
This PR styles the submission 'Thank you' page differently from the rest of the form.

It:
- doesn't render the navigation bar
- renders the page container as `display: block` instead of `display: grid` to have the contents span the full width of the page instead of falling in the grid's defined columns
- turns a regular text block into a heading
- cleans up unnecessary code from the `IncidentNavigation` component and replaces all of its tests

### CODE REVIEW

**Review by jpoppe — APPROVED** [review]

File: src/signals/incident/components/IncidentNavigation/index.test.js [hunk 1]
```diff
@@ -12,204 +15,124 @@ jest.mock('shared/services/auth/auth', () => ({
...
-  let withWizard;
-
-  function getComponent() {
-    wrapper = mount(
-      <Wizard history={historySpy}>
-        <IncidentNavigation {...props} />
-      </Wizard>
-    );
+const steps = Object.keys(wizardDefinition)
+  .filter(key => !['opslaan'].includes(key))
```
Comment (boris-arkenaar) [inline src/signals/incident/components/IncidentNavigation/index.test.js:19]: `  .filter(key => key !== 'opslaan')`

File: src/signals/incident/components/IncidentNavigation/index.test.js [hunk 2]
```diff
@@ -12,204 +15,124 @@ jest.mock('shared/services/auth/auth', () => ({
...
+            </Step>
+          </Steps>
+        </Wizard>
+      )
+    );
 
-      expect(wrapper).toMatchSnapshot();
-      expect(withWizardWrapper).toMatchSnapshot();
-    });
+    expect(getByTestId('nextButton')).toBeInTheDocument();
```
Comment (boris-arkenaar) [inline src/signals/incident/components/IncidentNavigation/index.test.js:49]: You could test here that the previous button does not exist.

File: src/signals/incident/components/IncidentNavigation/index.test.js [hunk 3]
```diff
@@ -12,204 +15,124 @@ jest.mock('shared/services/auth/auth', () => ({
...
+              <IncidentNavigation {...props} />
+            </Step>
+          </Steps>
+        </Wizard>
+      )
+    );
 
-      expect(props.meta.handleSubmit).toHaveBeenCalledWith(event, context.wizard.next, undefined);
-    });
+    expect(getByTestId('previousButton')).toBeInTheDocument();
```
Comment (boris-arkenaar) [inline src/signals/incident/components/IncidentNavigation/index.test.js:84]: Test here the next button does not exist.

File: src/signals/incident/components/IncidentNavigation/index.test.js [hunk 4]
```diff
@@ -12,204 +15,124 @@ jest.mock('shared/services/auth/auth', () => ({
...
+        </Wizard>
+      )
+    );
 
-      withWizardWrapper.find('[data-testid="nextButton"]').simulate('click', event);
+    expect(handleSubmit).not.toHaveBeenCalled();
 
-      expect(props.meta.handleSubmit).toHaveBeenCalledWith(event, context.wizard.next, 'CREATE_INCIDENT');
+    act(() => {
+      fireEvent.click(getByTestId('nextButton'));
```
Comment (boris-arkenaar) [inline src/signals/incident/components/IncidentNavigation/index.test.js:135]: You're missing a check that `handleSubmit` has been called after this statement.

**Review by boris-arkenaar — COMMENTED** [review]
Wouldn't it be better to make this thank you page a separate page instead of a step in the wizzard? Since now you have some very explicit logic in the code for making exceptions for that step.

**Review by boris-arkenaar — APPROVED** [review]
> Wouldn't it be better to make this thank you page a separate page instead of a step in the wizzard? Since now you have some very explicit logic in the code for making exceptions for that step.

What about this? ☝️

**Review by aditudorache — APPROVED** [review]

**Comment by janjaap** [comment]
> > Wouldn't it be better to make this thank you page a separate page instead of a step in the wizzard? Since now you have some very explicit logic in the code for making exceptions for that step.
> 
> What about this? ☝️

Agreed. But out of scope for this ticket.

**Pull request merged**